src/inverseRobot.js

```diff
@@ -24,7 +24,15 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reverseRobot = Object.fromEntries(Object.entries(robot).map(
+    ([key, value]) => [value, key]
+  ));
```

Comment:
🔥  need to format it a bit, tho
```suggestion
  const reverseRobot = Object.fromEntries(
    Object.entries(robot).map(
      ([key, value]) => [value, key]
    ),
  );
```